diff --git a/node_modules/did-jwt/lib/index.module.js b/node_modules/did-jwt/lib/index.module.js
index 50b7e0f..08f42ef 100644
--- a/node_modules/did-jwt/lib/index.module.js
+++ b/node_modules/did-jwt/lib/index.module.js
@@ -1662,7 +1662,7 @@ const resolveX25519Encrypters = function (dids, resolver) {
             const pks = (agreementKeys == null ? void 0 : agreementKeys.filter(key => {
               // TODO: should be able to use non base58 keys too
               return key.type === 'X25519KeyAgreementKey2019' && Boolean(key.publicKeyBase58);
-            })) ?? [];
+            })) || [];
             if (!pks.length && !controllerEncrypters.length) throw new Error(`no_suitable_keys: Could not find x25519 key for ${did}`);
             return pks.map(pk => x25519Encrypter(base58ToBytes(pk.publicKeyBase58), pk.id)).concat(...controllerEncrypters);
           }
